random.py

```diff
@@ -199,6 +199,15 @@ def generate_email(self):
     def run(self, view, **kwargs):
         self.insert(view, self.generate_email)
 
+class RandomHexColorCommand(RandomText):
+
+    def generate_hex_color(self):
+        #  Code taken from: http://stackoverflow.com/questions/13998901/generating-a-random-hex-color-in-python
+        return '#%06X' % random.randint(0,256**3-1)
```

Comment:
Thanks a lot for the PR!
I'm a bigger fan of: `"#%06x" % random.randint(0,0xFFFFFF)` it is a bit easier to understand. Would you mind changing it?